vncviewer/touch.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2019 Pierre Ossman <ossman@cendio.se> for Cendio AB
+ * 
+ * This is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * 
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this software; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307,
+ * USA.
+ */
+
+#ifndef __TOUCH_H__
+#define __TOUCH_H__
+
+#define Button6     6
+#define Button7     7
+
+#define Button6Mask (1<<13)
+#define Button7Mask (1<<14)
+
+void enable_touch();
+void disable_touch();
+
+int xi2_grabDevices(Window window);
+void xi2_ungrabDevices();
```

Comment:
Some nitpicking for the naming here:

 * We generally use `x11` for the naming and don't go in to specifics as to which extension is used. If nothing else it allows us to keep the name even if we start using other extensions.

 * You've got a weird mix of both underscore and uppercase. Pick one. :) I would suggest underscore as that's what we've mostly used for functions (and mixed case for classes and methods).

 * Should probably be "pointer" rather than "devices" as we don't grab everything.

Reply:
9fef2c4a